src/components/ContentNode.vue

```diff
@@ -297,18 +307,29 @@ function renderNode(createElement, references) {
         renderChildren(node.inlineContent)
       ));
     }
-    case BlockType.table:
+    case BlockType.table: {
+      let tableContent = [renderTableChildren(
```

Comment:
I dont think you need to wrap this in an array 🤔

Reply:
True, I left it behind after a refactor. I deleted it now